rome-modules/src/main/java/com/rometools/modules/itunes/types/Category.java

```diff
@@ -64,7 +67,20 @@ public void setName(final String name) {
      * @return Returns the Subcategory object for this category
      */
     public Subcategory getSubcategory() {
-        return subcategory;
+        return subcategories.get(0);
```

Comment:
Backward incompatible change: if there are no subcategories it used to return null, now it's throwing null pointer.